packages/backend/src/index.ts

```diff
@@ -210,7 +210,7 @@ app.use((error: Error, req: Request, res: Response, _: NextFunction) => {
 });
 
 // Run the server
-const port = process.env.PORT || 8080;
+const port = process.env.SERVER_PORT || 8080;
```

Comment:
Any reason to make this breaking change ?

Reply:
I think it was breaking frontend. I'll double check and revert if it's not necesary

Reply:
yeah, so webpack picks up PORT env variable and spawns its server with 8080. I will revert this once we resolve #3257